src/routes/expenses.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
```

Comment:
Replace 'use strict' with ECMAScript modules syntax by using 'import' instead of 'require'.